recipes/apache-airflow-providers-microsoft-azure/0001-remove-azure-keyvault-metapackage.patch

```diff
@@ -0,0 +1,11 @@
+diff -ruN apache-airflow-providers-microsoft-azure-1.1.0/setup.py apache-airflow-providers-microsoft-azure-1.1.0-patch/setup.py
+--- setup.py	2021-02-08 21:36:01.000000000 +0100
++++ setup.py	2021-02-24 08:51:49.194796581 +0100
+@@ -63,7 +63,6 @@
+             'azure-cosmos>=3.0.1,<4',
+             'azure-datalake-store>=0.0.45',
+             'azure-identity>=1.3.1',
+-            'azure-keyvault>=4.1.0',
```

Comment:
On conda-forge, this is a matapackage that doesn't provide the info that `pip check` expects.  It's part of the recipe but I'm taking it out of `setup.py` so `pip check` is happy.

Reply:
Should we add this as a real package then?

Reply:
It certainly seems like packages that are on pypi but which have been made into conda metapackages on `conda-forge` are causing trouble for `pip check` and a general solution to these situations is going to be needed.  Patching upstream packages to remove the dependencies isn't ideal.

Reply:
I made an issue to track this.  
https://github.com/conda-forge/azure-keyvault-feedstock/issues/6
I'll remove this patch as soon as there's an alternative solution.